src/storage/src/hummock/sstable/builder.rs

```diff
@@ -76,6 +79,7 @@ impl Default for SstableBuilderOptions {
             bloom_false_positive: DEFAULT_BLOOM_FALSE_POSITIVE,
             compression_algorithm: CompressionAlgorithm::None,
             max_key_count: DEFAULT_MAX_KEY_COUNT,
+            max_sst_size: DEFAULT_MAX_SST_SIZE,
```

Comment:
minor: SstableBuilderOptions::default is only used in test. I think make it cfg(test) and as well as all DEFAULT_* can help avoid misused.